OpenSim/Simulation/Model/Appearance.h

```diff
@@ -59,8 +59,8 @@ class OSIMSIMULATION_API SurfaceProperties : public Object {
     //==========================================================================
     OpenSim_DECLARE_PROPERTY(representation, int,
      "The representation (1:Points, 2:Wire, 3:Shaded) used to display the object.");
-    OpenSim_DECLARE_OPTIONAL_PROPERTY(texture_file, std::string,
-        "Name of file containing texture.");
+    OpenSim_DECLARE_OPTIONAL_PROPERTY(texture, std::string,
+        "Name of texture e.g. metal, bone. This is a hint to the GUI/Visualizer, implementation is Visualization dependent.");
```

Comment:
Can we list the textures that are supported by the standard OpenSim visualizer, or tell users how to find this list?